client.lua

```diff
@@ -235,11 +258,189 @@ CreateThread(function()
     while true do
         Wait(1000)
         if LocalPlayer.state.isLoggedIn and onRadio then
-            if not hasRadio or PlayerData.metadata.isdead or PlayerData.metadata.inlaststand then
+            if not hasRadio or PlayerData.metadata.isdead then
                 if RadioChannel ~= 0 then
                     leaveradio()
                 end
             end
         end
     end
 end)
+
+-- Leave Radio
+RegisterNetEvent('qb-radio:client:leaveradio', function()
+    leaveradio()
+end)
+
+RegisterNetEvent('qb-radio:client:volupradio', function(val)
+    local newvalue = tonumber(val)
+    if newvalue <= 100  and newvalue >= 0 then
+        RadioVolume = newvalue
+		QBCore.Functions.Notify(Config.messages["volume_radio"] .. RadioVolume, "success")
+		exports["pma-voice"]:setRadioVolume(RadioVolume)
+	else
+		QBCore.Functions.Notify("Invalid Volume Amount", "error")
+	end
+end)
+
+RegisterNetEvent('qb-radio:client:JoinRadioChannel1', function()
```

Comment:
Large function repeated here means more code to test / maintain.
Please consider making the channel id something which can be passed in